### PR TITLE
Added more paths to coverage path combiner

### PR DESCRIPTION
**Pull request by alex**

### CODE REVIEW

File: .coveragerc [hunk 1]
```diff
@@ -8,6 +8,8 @@ source =
 source =
    src/cryptography
    .tox/*/lib/python*/site-packages/cryptography
+   .tox/*/lib64/python*/site-packages/cryptography
```
Comment (reaperhulk) [inline .coveragerc:11]: Could we just do lib*?

Comment (alex) [reply .coveragerc:11]: We can, would you prefer that?

Comment (reaperhulk) [reply .coveragerc:11]: Please

Comment (alex) [reply .coveragerc]: done

**Review by reaperhulk — APPROVED** [review]

**Pull request merged**